app/serializers/user_serializer.rb

```diff
@@ -282,10 +282,7 @@ def ignored
   end
 
   def can_ignore_user
-    SiteSetting.ignore_user_enabled? &&
-      !object.staff? &&
-      scope.current_user != object &&
-      (scope.current_user.staff? || scope.current_user.trust_level >= TrustLevel.levels[:member])
+    SiteSetting.ignore_user_enabled? && scope.can_ignore_user?(object.id)
```

Comment:
Should we move that check in the `guardian.can_ignore_users?` method?

Reply:
We can I think. I don't see much strong pros or cons out of moving it.